build_runner/pubspec.yaml

```diff
@@ -57,3 +57,6 @@ dev_dependencies:
   test: ^1.16.0
   test_descriptor: ^2.0.0
   test_process: ^2.0.0
+
+dependency_overrides:
+  frontend_server_client: 4.0.0
```

Comment:
Looks like the tests passed, so now we should remove this before landing :). But otherwise LGTM

```suggestion
```